API/common/utils.py

```diff
@@ -58,7 +59,8 @@ def execute(cwd, cmd, args=None, quiet=False):
         return output, exitcode
 
     except Exception as e:
-        console.fail('[Failed - %s %s] %s' % (cmd, ' '.join(args), str(e)))
+        console.log('[Failed - %s %s] %s' % (cmd, ' '.join(args), str(e)), '\033[1;31m')
+        sys.exit(1)
```

Comment:
I think you should use `console.fail` here and replace the `exit(1)` to `sys.exit(1)` in `console.fail`.